app/org/sagebionetworks/bridge/json/DateTimeToPrimitiveLongDeserializer.java

```diff
@@ -0,0 +1,36 @@
+package org.sagebionetworks.bridge.json;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+/**
+ * The normal DateTimeJsonDeserializer will treat JSON empty and null values as null Longs. But when 
+ * these are sent for fields that are primitive longs, a JSON deserialization exception occurs. 
+ * To the JSON producer, this makes no sense (it's strictly because of the internal Java type system). 
+ * 
+ * This deserializer uses the default primitive long value (0L) which does not throw an exception; the 
+ * system should validate that this value is acceptable. In our system, in all places where we set 
+ * primitive long timestamps, values from the client are ignored and instead set on the server. 
+ */
+public class DateTimeToPrimitiveLongDeserializer extends JsonDeserializer<Long> {
+
+    @Override
+    public Long deserialize(JsonParser jp, DeserializationContext dc) throws IOException, JsonProcessingException {
+        String date = jp.getText();
+        return DateUtils.convertToMillisFromEpoch(date);
+    }
+    
+    @Override
+    public Long getNullValue() {
```

Comment:
0 can be a perfectly valid value which can be the Unix epoch at 1970-01-01T00:00:00Z. Are we introducing the risk here that null is turned into something meaningful.  Like my DOB is "unknown" and after deserialized it becomes 1970/01/01.